### PR TITLE
pt-br translation for forks PSA

### PR DESCRIPTION
**Pull request by netrik182**

### CODE REVIEW

**Comment by lh1008** [comment]
Thanks @netrik182, pinging @botoserelepe and/or @rating89us for review. :)

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Conheça seus hard forks
+
+A não ser que você seja desenvolvedor, o termo "hard fork" pode causar muita confusão e potencialmente trazer conotações negativas para você. Afinal, você já deve ter lido um artigo ou outro sobre hard forks e como ele causou contenção ou drama em alguns projetos de cripto. Os hard forks do Monero acontecem de maneira interessante e enquadram-se em três categorias principais:
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]:
```suggestion
A não ser que você seja desenvolvedor, o termo "hard fork" pode causar muita confusão e eventualmente trazer conotações negativas. Afinal, você já deve ter lido um ou outro artigo sobre hard forks e como eles causaram divisões ou conflitos em alguns projetos de cripto. Os hard forks do Monero acontecem de maneira interessante e enquadram-se em três categorias principais:
```

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### 1) Atualização da rede / Hard fork regularmente programado
+
+Os desenvolvedores do Monero têm por objetivo realizar dois hard forks regulares por ano. Esses hard forks são sempre feitos com o consenso da comunidade e liderado pelos Desenvolvedores Core. Eles geralmente incluem diversas melhorias ao código principal do Monero. É necessário que você, como usuário Monero, atualize suas carteiras sempre que isso ocorrer e que não é responsabilidade de ninguém, senão você mesmo, de fazê-lo. Ninguém disse que seria fácil ser um pioneiro da tecnologia, mas se for reparar bem, não é tão difícil também.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]:
```suggestion
Os desenvolvedores do Monero têm por objetivo realizar regularmente dois hard forks por ano. Esses hard forks são sempre feitos com o consenso da comunidade e liderado pelos Desenvolvedores Core. Eles geralmente incluem diversas melhorias no código principal do Monero. É necessário que você, como usuário Monero, atualize suas carteiras sempre que isso ocorrer. E não é responsabilidade de ninguém, a não ser de você mesmo, de fazê-lo. Ninguém disse que seria fácil ser um pioneiro da tecnologia, mas se for reparar bem, não é tão difícil também.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Outros projetos utilizaram os hard forks regularmente programados do Monero para seu próprio benefício e criaram seus próprios projetos sobre as cadeias _(chains)_ anteriores do Monero. Esses projetos costumam fracassar rapidamente, levando consigo a ganância dos seus criadores. Se você não atualizar sua carteira Monero após um desses hard forks e transferir seus XMR, você pode acabar perdendo-os em uma _chain_ diferente ou gastar um tempo considerável tentando recuperá-los.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: On the second line, I would just put the word "chains" in italic: (_chains_)
Suggestion: "Outros projetos se aproveitaram dos hard forks regularmente programados do Monero e criaram [...]" Check which one you prefer.

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+_Nível de segurança:_ A melhor prática é atualizar suas carteiras Monero assim que atualizações forem lançadas.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: Maybe it is interesting to add "atualizações de hard forks" on the end. So it enphasizes which updates you are talking about.

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+
+Alguns projetos legítimos são derivados de hard forks do Monero, utilizando seu código base para alcançar seus objetivos. Existem alguns fatores que determinam se um projeto é legítimo:
+
+- Não há pré-mineração ou mineração instantânea de moeadas para os desenvolvedores.
+- Não é exigido que você compartilhe suas chaves privadas do Monero.
+- Não há avaliações controversas, excessivamente negativas ou alarmantes sobre o projeto na comunidade Monero.
+- O projeto é código aberto e faz doações ao Monero.
+
+_Nível de segurança:_ Se o projeto é legítimo, você não precisará fazer nada com sua carteira Monero e você pode se juntar ao projeto se assim o desejar.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: Suggestion:  "Se o projeto for realmente legítimo"

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+### 3) Hard forks de golpe ou enganosos
+
+Esses projetos são o oposto dos projetos legítimos mencionados acima. Esses projetos são movidos pela ganância, tanto por parte dos desenvolvedores como pelos usuários (lê-se vítimas). Em alguns casos, especialmente nos projetos em que se pede aos usuários para compartilharem suas chaves privadas, eles podem ser uma tentativa de fraude para hackear a rede Monero e fragilizar a privacidade/anonimato de seus usuários.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]:
```suggestion
Esses projetos são o oposto dos projetos legítimos mencionados acima. Esses projetos são movidos pela ganância, tanto por parte dos desenvolvedores como pelos usuários (lê-se vítimas). Em alguns casos, especialmente naqueles em que é pedido aos usuários para compartilharem suas chaves privadas, eles caracterizam uma tentativa de fraude para hackear a rede Monero e põem em risco a privacidade ou anonimato de seus usuários.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+_Nível de segurança:_ É fortemente recomendado que você evite essas moedas. Prossiga por sua conta e risco e esteja ciente de que sua participação pode inadvertidamente enfraquecer a rede Monero.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]:
```suggestion
_Nível de segurança:_ É fortemente recomendado que você evite essas moedas. Prossiga por sua conta e risco e esteja ciente de que sua participação pode inadvertidamente debilitar a rede Monero.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md (hunk 3 above)
Comment (netrik182) [reply monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: Thanks for the suggestion @botoserelepe

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md (hunk 4 above)
Comment (netrik182) [reply monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: I would say that users should update their software either being it due a hard fork or not. What do you think?

Comment (botoserelepe) [reply monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: Agreed.

File: monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
+A não ser que você seja desenvolvedor, o termo "hard fork" pode causar muita confusão e eventualmente trazer conotações negativas. Afinal, você já deve ter lido um ou outro artigo sobre hard forks e como eles causaram divisões ou conflitos em alguns projetos de cripto. Os hard forks do Monero acontecem de maneira interessante e enquadram-se em três categorias principais:
 
 ### 1) Atualização da rede / Hard fork regularmente programado
 
-Os desenvolvedores do Monero têm por objetivo realizar dois hard forks regulares por ano. Esses hard forks são sempre feitos com o consenso da comunidade e liderado pelos Desenvolvedores Core. Eles geralmente incluem diversas melhorias ao código principal do Monero. É necessário que você, como usuário Monero, atualize suas carteiras sempre que isso ocorrer e que não é responsabilidade de ninguém, senão você mesmo, de fazê-lo. Ninguém disse que seria fácil ser um pioneiro da tecnologia, mas se for reparar bem, não é tão difícil também.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: I haven't seen this one: "liderado" on the second line must be in plural: "liderados"

Comment (netrik182) [reply monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md]: Good catch. Thanks again @botoserelepe

**Pull request merged**